x-pack/plugins/lens/public/indexpattern_datasource/indexpattern.tsx

```diff
@@ -663,6 +665,56 @@ export function getIndexPatternDatasource({
         ),
       ];
     },
+    getDeprecationMessages: () => {
+      const deprecatedMessages: React.ReactNode[] = [];
+      const useFieldExistenceSamplingKey = 'lens:useFieldExistenceSampling';
+      const isUsingSampling = core.uiSettings.get(useFieldExistenceSamplingKey);
+
+      if (isUsingSampling) {
+        const link = (
+          <EuiLink
+            color="accent"
```

Comment:
Why the use of `accent` color here? Can we not use the default `primary` color? 

Alternatively, if the `primary` link color looks awful, we can also try using the `warning` color, but add custom styling to have it always appear underlined.